.gitignore

```diff
@@ -32,3 +32,8 @@ dockstore-webservice/nbactions.xml
 # confidential testing files
 secrets.tar
 dockstore-integration-testing/src/test/resources/confidential_test.txt
+dockstore-integration-testing/src/test/resources/config_file.txt
+dockstore-integration-testing/src/test/resources/db_dump.sql
```

Comment:
Can you add a few comments here why both the db_dump and the confidential dump are ignored? Intuitively, I would have thought that only the latter should be ignored.